src/actions/expressions.js

```diff
@@ -90,7 +90,7 @@ export function deleteExpression(expression: Expression) {
  * @param {number} selectedFrameId
  * @static
  */
-export function evaluateExpressions(frameId: frameIdType) {
+export function evaluateExpressions(frameId?: frameIdType) {
```

Comment:
Its better if we keep evaluate expression to having just one responsibility. It looks like up to this point we always had a frameID -- if this is true, then we can remove the if block here